requirements/pip.txt

```diff
@@ -52,7 +55,7 @@ stripe==1.20.2
 django-copyright==1.0.0
 django-formtools==1.0
 django-dynamic-fixture==1.8.5
-docker-py==1.3.1
+docker==2.6.1
```

Comment:
`2.7.0` is out

Reply:
from its changelog it seems to be safe to use it https://docker-py.readthedocs.io/en/stable/change-log.html#

besides, I just tested it locally and it just works